DQM/HcalMonitorTasks/src/HcalDetDiagNoiseMonitor.cc

```diff
@@ -633,7 +633,8 @@ void HcalDetDiagNoiseMonitor::done(){}
  
 HcalDetDiagNoiseMonitor::~HcalDetDiagNoiseMonitor()
 {
-  if(LocalRun) UpdateHistos(); SaveRates(); 
```

Comment:
SaveRates is used elsewhere for !LocalRun so I assume this should not be enclosed in curly braces.